api/src/main/java/io/smallrye/reactive/messaging/annotations/Incomings.java

```diff
@@ -8,13 +8,37 @@
 import org.eclipse.microprofile.reactive.messaging.Incoming;
 
 /**
- * Container annotation to declare several {@link Incoming}.
+ * This class is used to allow multiple {@link Incoming} declarations.
+ * You can either use:
+ *
+ * <pre><code>
+ * &#64;Incoming("a")
+ * &#64;Incoming("b")
+ * public void consume(T t);
+ * </code></pre>
+ *
+ * Or use the {@link Incomings} annotation as container:
```

Comment:
so we're saying it's ok to use `@Incomings` now?

For some reason I thought the resolution was to javadoc not to use it? But I may have been distracted ;-)

Reply:
:-D

I just follow what other specs have done, and let the user decide which flavor he prefers. 
See https://javaee.github.io/javaee-spec/javadocs/javax/annotation/Resources.html, too succinct for my taste, and does not say anything about the usage. 

So as we can't prevent the user to use it, let's explain how.

Reply:
fair enough